chart/kubeapps/templates/tests/test-chartsvc.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ .Release.Name }}-chartsvc-test"
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+  - name: {{ .Release.Name }}-chartsvc-test
+    image: {{ .Values.frontend.image.repository }}:{{ .Values.frontend.image.tag }}
+    env:
+      - name: CHARTSVC_HOST
```

Comment:
indentation is mismatched here compared to the containers array, we should have `-` aligned with `name:` above